File: backend-python/app/modules/system/entity/sys_config.py
from sqlalchemy import BigInteger, Integer, String
from sqlalchemy.orm import Mapped, mapped_column

from app.modules.system.entity.base import Base, TimestampMixin


class SysConfig(TimestampMixin, Base):
    __tablename__ = "sys_config"

    id: Mapped[int] = mapped_column(BigInteger, primary_key=True)
    config_name: Mapped[str] = mapped_column(String(64))
    config_key: Mapped[str] = mapped_column(String(128))
    config_value: Mapped[str] = mapped_column(String(1024))
    value_type: Mapped[str] = mapped_column(String(32))
    builtin: Mapped[int] = mapped_column(Integer)
    status: Mapped[int] = mapped_column(Integer)
    remark: Mapped[str | None] = mapped_column(String(255), nullable=True)
    deleted: Mapped[int] = mapped_column(Integer)

File: backend-go/internal/modules/system/entity/sys_config.go
package entity

import (
	"drip-admin/backend-go/internal/common"
	"time"
)

type SysConfig struct {
	ID          common.Int64String `gorm:"column:id;primaryKey" json:"id"`
	ConfigName  string             `gorm:"column:config_name" json:"configName"`
	ConfigKey   string             `gorm:"column:config_key" json:"configKey"`
	ConfigValue string             `gorm:"column:config_value" json:"configValue"`
	ValueType   string             `gorm:"column:value_type" json:"valueType"`
	Builtin     int                `gorm:"column:builtin" json:"builtin"`
	Status      int                `gorm:"column:status" json:"status"`
	Remark      *string            `gorm:"column:remark" json:"remark"`
	Deleted     int                `gorm:"column:deleted" json:"deleted"`
	CreatedAt   *time.Time         `gorm:"column:created_at" json:"createdAt"`
	UpdatedAt   *time.Time         `gorm:"column:updated_at" json:"updatedAt"`
}

func (SysConfig) TableName() string { return "sys_config" }
